routers/user.py

```diff
@@ -0,0 +1,259 @@
+import httpx
+
+from auth.config import get_settings
+from auth.management import get_management_token
+from auth.validator import get_current_user
+from datetime import datetime, timezone
+from fastapi import APIRouter, Depends, HTTPException
+from schemas.requests import ResourceRequest, ServiceRequest
+from schemas.user import User
+
+router = APIRouter()
+
+def get_settings_instance():
+    return get_settings()
+
+async def fetch_user_data(user_id: str, token: str):
```

Comment:
Can we create a schema/Pydantic model for what the user data should look like, and return the model? Makes it easier to use the data when you're writing routes that use it.